bakerydemo/static/css/main.css

```diff
@@ -1386,9 +1401,10 @@ input[type="radio"] {
     margin: unset;
   }
 
-  .featured-cards__chevron-icon {
-    width: 11px;
-    height: 30px;
+  .homepage .featured-cards__chevron-icon {
```

Comment:
question: now this has the `.homepage` selector in front are there non homepage `.featured-cards__chevron-icon`'s that will loose styling or is it set elsewhere?

Reply:
There are no `featured-cards` sections used outside of the homepage so this is OK as is